blazor/rich-text-editor/how-to/realtime-editing-signalR.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: Realtime Editing with SignalR in RichTextEditor | Syncfusion
+description: Learn here all about renaming images before inserting it in Syncfusion Blazor RichTextEditor component and more.
+platform: Blazor
+control: RichTextEditor
+documentation: ug
+---
+
+# Realtime editing with SignalR
+
+The Rich Text Editor allows, two or more users can edit the same editor by using [SignalR](https://docs.microsoft.com/en-us/aspnet/core/tutorials/signalr?view=aspnetcore-6.0&tabs=visual-studio) services. Which will update the editor value in real-time, when loading it on multiple pages. In the below example create the `SignalR HubConnectionBuilder` in the `OnInitializedAsync` method and the editor updated value send to the SignalR message in the [ValueChange](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.RichTextEditor.RichTextEditorEvents.html#Syncfusion_Blazor_RichTextEditor_RichTextEditorEvents_ValueChange) event.
```

Comment:
The Rich Text Editor allows, two or more users can edit the same editor by using -> Did we checked grammer?